src/Xamarin.Android.Build.Tasks/Tasks/AssemblyModifierPipeline.cs

```diff
@@ -20,16 +20,8 @@ namespace Xamarin.Android.Tasks;
 /// are run *after* the linker has run. Additionally, this task is run by
 /// LinkAssembliesNoShrink to modify assemblies when ILLink is not used.
 /// </summary>
-public class AssemblyModifierPipeline : AndroidTask
+public partial class AssemblyModifierPipeline : AndroidTask
 {
-	// Names of assemblies which don't have Mono.Android.dll references, or are framework assemblies, but which must
-	// be scanned for Java types.
-	static readonly HashSet<string> SpecialAssemblies = new HashSet<string> (StringComparer.OrdinalIgnoreCase) {
```

Comment:
How many "copies" of this list do we *have*?!

See also `MonoAndroidHelper.Linker.cs` which has `MonoAndroidHelper.KnownAssemblyNames`, which includes:

  * `Mono.Android`
  * `Mono.Android.Export`
  * `Java.Interop`

The ones that "don't match" are `Mono.Android.Export` and `Mono.Android.Runtime`, which is probably more an oversight than intentional…

Reply:
> Why have/use `SpecialAssembles` instead of `KnownAssemblyNames`, in `MonoAndroidHelper.Linker.cs`?

The unsatisfying answer is I simply moved the existing logic.😉 

> The ones that "don't match" are `Mono.Android.Export` and `Mono.Android.Runtime`, which is probably more an oversight than intentional…

Maybe? We would likely need to audit all users of both lists to see what their intentions are.

Reply:
Unified the lists and CI isn't complaining, so I guess that's good enough! 🤷‍♂️